SPECS/softhsm/softhsm.spec

```diff
@@ -1,29 +1,24 @@
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
-#global prever rc1
-#global prerelease yes
-
-Summary: Software version of a PKCS#11 Hardware Security Module
-Name: softhsm
-Version: 2.6.1
-Release: 5%{?dist}
-License: BSD
-Url: http://www.opendnssec.org/
-Source: http://dist.opendnssec.org/source/%{?prever:testing/}%{name}-%{version}.tar.gz
-Source1: http://dist.opendnssec.org/source/%{?prever:testing/}%{name}-%{version}.tar.gz.sig
-
-Patch1: softhsm-2.6.1-rh1831086-exit.patch
-
-BuildRequires: openssl-devel >= 1.0.1k-6, sqlite-devel >= 3.4.2, cppunit-devel
-BuildRequires: gcc-c++, pkgconfig, p11-kit-devel
-
-Requires(pre): shadow-utils
-Requires: p11-kit
-Requires: openssl-libs >= 1.0.1k-6
-
 %global _hardened_build 1
-
 %global softhsm_module "SoftHSM PKCS #11 Module"
+Summary:        Software version of a PKCS#11 Hardware Security Module
+Name:           softhsm
+Version:        2.6.1
+Release:        6%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://www.opendnssec.org/
+Source0:        https://dist.opendnssec.org/source/%{?prever:testing/}%{name}-%{version}.tar.gz
```

Comment:
In addition to removing the commented `prerelease` and `prever` macros, please also remove their usage in the spec.